### ISSUE TITLE
Wrong behavior with two and more decorators

### ISSUE THREAD

**Issue by vovanbo** [Helioscene/dry-rest-permissions#6]
Code below throw AttributeError, because of generic decorators use function name to set `is_object_permission`:
```python
@authenticated_users
@allow_staff_or_superuser
def has_object_read_permission(self, request):
    return False
```

**Comment by dbkaplan** [comment]
Thanks for finding this. I see the issue and will try to get it fixed today.

**Comment by vovanbo** [comment]
I've already fix it in PR #7. I had try to make tests for it, but it seems complicated for me now and i have no time to go deeper in this test case ;)
